runtime/gc_include/ObjectAllocationAPI.hpp

```diff
@@ -58,6 +58,67 @@ class MM_ObjectAllocationAPI
 	const J9VMGCSizeClasses *_sizeClasses;
 #endif /* J9VM_GC_SEGREGATED_HEAP */
 
+	VMINLINE void
+	initializeIndexableSlots(bool initializeSlots, UDATA dataSize, UDATA headerSize, J9IndexableObject *header)
```

Comment:
minor difference, but you can just pass dataAddr value, instead of passing headerSize and header